src/systems/systems/src/navigation/radio_altimeter.rs

```diff
@@ -0,0 +1,864 @@
+                radio_altimeter_1: Ala52BRadioAltimeter::new(
+                    context,
+                    1,
+                    Ala52BAircraftInstallationDelay::FiftySevenFeet,
+                    ElectricalBusType::AlternatingCurrent(1),
+                ),
+                // roughly model the A320 RA 1 transceivers
+                system_1_transceivers: Ala52BTransceiverPair::new(
+                    context,
+                    AntennaInstallation::new(
+                        Length::new::<foot>(8.617) - Length::new::<meter>(1.8),
+                        Length::new::<foot>(9.89),
+                        Length::new::<foot>(22.6),
+                    ),
+                    AntennaInstallation::new(
+                        Length::new::<foot>(8.617) - Length::new::<meter>(1.8),
+                        Length::new::<foot>(9.19),
+                        Length::new::<foot>(22.6),
+                    ),
+                ),
+                is_ac_1_powered: false,
+                power_consumption: Power::new::<watt>(0.),
+            }
+        }
+
+        fn radio_altimeter(&self, number: usize) -> &Ala52BRadioAltimeter {
+            match number {
+                1 => &self.radio_altimeter_1,
+                _ => panic!("The TestAircraft only has one radio altimeter."),
+            }
+        }
+
+        fn set_ac_1_power(&mut self, is_powered: bool) {
+            self.is_ac_1_powered = is_powered;
+        }
+
+        fn power_consumption(&self) -> Power {
+            self.power_consumption
+        }
+    }
+    impl Aircraft for TestAircraft {
+        fn update_before_power_distribution(
+            &mut self,
+            _: &UpdateContext,
+            electricity: &mut Electricity,
+        ) {
+            self.electricity_source
+                .power_with_potential(ElectricPotential::new::<volt>(115.));
+            electricity.supplied_by(&self.electricity_source);
+
+            if self.is_ac_1_powered {
+                electricity.flow(&self.electricity_source, &self.ac_1_bus);
+            }
+        }
+
+        fn update_after_power_distribution(&mut self, context: &UpdateContext) {
+            self.radio_altimeter_1
+                .update(context, &self.system_1_transceivers);
+        }
+    }
+    impl SimulationElement for TestAircraft {
+        fn process_power_consumption_report<T: PowerConsumptionReport>(
+            &mut self,
+            _: &UpdateContext,
+            report: &T,
+        ) {
+            self.power_consumption =
+                report.total_consumption_of(PotentialOrigin::EngineGenerator(1));
+        }
+
+        fn accept<T: SimulationElementVisitor>(&mut self, visitor: &mut T) {
+            self.system_1_transceivers.accept(visitor);
+            self.radio_altimeter_1.accept(visitor);
+
+            visitor.visit(self);
+        }
+
+        fn read(&mut self, _reader: &mut SimulatorReader) {}
+
+        fn write(&self, _writer: &mut SimulatorWriter) {}
```

Comment:
**suggestion**: As default empty implementation exists for these. There's no need to repeat it here.
```suggestion
```